validphys2/src/validphys/loader.py

```diff
@@ -674,10 +674,10 @@ def check_default_filter_rules(self, theoryid, defaults=None):
         th_params = theoryid.get_description()
         if defaults is None:
             defaults = default_filter_settings_input()
-        return [
-            Rule(inp, defaults=defaults, theory_parameters=th_params, loader=self)
+        return tuple(
+            Rule(inp.to_dict(), defaults=defaults, theory_parameters=th_params, loader=self)
```

Comment:
```suggestion
            Rule(inp, defaults=defaults, theory_parameters=th_params, loader=self)
```

(same as above, I think it'd be better to have `Rule` take the dataclass by default and convert it to a dict inside)